cfgov/ask_cfpb/views.py

```diff
@@ -163,7 +163,7 @@ def redirect_ask_search(request, language='en'):
     else:
         facets = request.GET.getlist('selected_facets')
         if not facets or not facets[0]:
-            raise Http404
+            return redirect('/ask-cfpb/search')
```

Comment:
@richaagarwal or @higs4281 Do you know if there was a specific reason that this was set to raise a 404 instead of redirecting to the basic search page with no query? Any objection to changing that? (Here and L221 below.)

```suggestion
            return redirect('/ask-cfpb/search/')
```

@cwdavies Adding the trailing slash will save the app the work of adding it after the request (here and L221 below).